app/views/api/v1/crops/show.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.data do
+  json.(@crop, :id, :name, :creator_id, :en_wikipedia_url)
+  json.set! '@id', url_for(:only_path => false)
```

Comment:
this is inconsistent with the @id/url given in index.json.builder... which do we want?